src/cap-description.adoc

```diff
@@ -179,16 +179,27 @@ references if that SDP bit is set because it "owns" that object.
 ==== Sealed (S) Bit
 
 ifdef::cheri_v9_annotations[]
-WARNING: *CHERI v9:* The sealing bit is new (1-bit otype) and the old CHERI v9 otype no longer exists.
+WARNING: *CHERI v9:* The sealing bit is new (1-bit otype) and the old CHERI v9
+otype no longer exists. Please note that this bit indicates the result of what
+was done in two instructions in CHERI v9: CSeal for sealed capabilities and
+CSealEntry for sealed entry capabilties.
 endif::[]
 
-Indicates that a capability is sealed if the bit is 1 or unsealed if it is 0.
+This bit indicates that a capability is sealed if the bit is 1 or unsealed if it is 0.
+
+The sealing bit conflates two concepts in one bit: Sealing data capabilities and
+creating sealed entry (sentry) capabilities.
```

Comment:
I don't recall the concept of a "sealed entry" explained in this spec. Could you write a couple of sentences elaborating on this?